.cloudbees/testing/action.yml

```diff
@@ -27,6 +27,11 @@ inputs:
     description: >
       Registry mirror(s) to use for loading images.
       Type: CSV
+  registry-maps:
+    description: >
+      Registry map(s) to remap registries references.
+      To specify several registry mappings separate them by semi-colon. eg. index.docker.io=docker-io.mirrors.corp.net;index.docker.io=mirror.gcr.io;gcr.io=127.0.0.1
+      By default it uses the registries configured under 'Integrations' in the CloudBees platform.
```

Comment:
Why introduce this new option when we have the platform-wide registry mirror configuration now? Do we have a concrete use-case for that?
Either way, to make it clear that/how the platform registry mirror configuration is consumed by this Action, I recommend exposing an input to provide the path to a CloudBees registry mirror configuration file and let that input default to `${{cloudbees.registries}}`. WDYT?

Reply:
> Why introduce this new input option when we have the platform-wide registry mirror configuration now? Do we have a concrete use-case for that?

I thought it provides more flexibility, where if a user does not find a particular mirror configured in the platform and the user does not have permissions to add the mirror under the Org or Sub-Org, they are not blocked and can use this option provided by the action.

Reply:
Sounds like a very edgy case that does not exist in reality. Users can configure registry mirrors within the platform via its GUI. If that case would exist, they could still unblock themselves by adding their additional registry mirror to the CloudBees `registries.json` file (e.g. using `jq`). Thus, even if that edgy use-case exists, there would be a way for users to unblock themselves via the CloudBees `registries.json` configuration file.
That said, I'd like us to only add to public interfaces what's really required as opposed to what might be nice-to-have but is technically not necessary. That is because we have to maintain every single one of those inputs over time and keep them backward-compatible and the fewer inputs there are the faster users can skim over them to find the input they need.

Reply:
Ok, I will remove the option for users to specify registry mappings through an input in this action and just process what is defined in the platform and incorporated in the CloudBees registry config JSON.